### DISCUSSION TITLE
Wall Boundary Conditions around a cylinder

### DISCUSSION THREAD

**Discussion by giakbottacini** [lululxvi/deepxde#1763] [Q&A]
Hi, 
I'm solving the compressible Euler equations around a cylinder in a 2D domain. I need to impose the normal velocity at the cylinder equal to 0, is it possible using deepxde?
For now i impose an homogeneous Dirichlet in this way 
bc_c1 = dde.icbc.DirichletBC(spatial_domain, lambda x: 0, boundary_cylinder, component=0)
bc_c2 = dde.icbc.DirichletBC(spatial_domain, lambda x: 0, boundary_cylinder, component=1)
on both horizontal and vertical velocity but is not right in a case of zero viscosity as this.
Thank you.
Giacomo

**Comment by lululxvi** [comment]
Seems like Neumann BC.
